### ISSUE TITLE
Move Windpowerlib to oemof community

### ISSUE THREAD

**Issue by p-snft** [oemof/oemof#92]
At https://github.com/oemof/oemof#contents, windpowerlib is counted as an oemof project. However, it is located at https://github.com/wind-python/windpowerlib, while all other projects are sitting within https://github.com/oemof/. I think, it should be moved here.

**Comment by simnh** [comment]
I see your point. However, I also think that it might be valuable if specific wind (or hydro, etc.) communities on github that can have their own places. I am not sure how to deal with it. Ideally, it would be "located" at both similar a mirror or  symbolic link. 

Once wind-python has 10 repositories these can not be added to oemof, and then the question is why the windpowerlib ist not there. Don't get me wrong, I like your idea but am unsure on how to proceed within a growing framework with increasing number of packages and tools etc. At some point we might need to think about it...(maybe)

**Comment by p-snft** [comment]
You can have mirrors but mirroring from GitHub to Github is a bit awkward.

At the moment, there is even a repository [oemof/tespy.extensions](https://github.com/oemof/tespy.extensions), while at the same time, there is the windpowerlib space with just one repository. (I do not count the empty [synthetic_power_curves](https://github.com/wind-python/synthetic_power_curves) one.) Maybe, this is a topic for the next meeting.

**Comment by p-snft** [comment]
Also relates to #91.

**Comment by fwitte** [comment]
> At the moment, there is even a repository [oemof/tespy.extensions](https://github.com/oemof/tespy.extensions)

Yeah... We had a plan about that :D. I'll deactivate it for the time being.

But generally, I think having windpowerlib and hydropowerlib inside the oemof organization might make sense, too.
